cmd/restic/cmd_dump.go

```diff
@@ -168,7 +165,24 @@ func runDump(ctx context.Context, opts DumpOptions, gopts GlobalOptions, args []
 		return errors.Fatalf("loading tree for snapshot %q failed: %v", snapshotIDString, err)
 	}
 
-	d := dump.New(opts.Archive, repo, os.Stdout)
+	var outputWriter io.Writer = os.Stdout
+	if opts.Target != "" {
+		file, err := os.OpenFile(opts.Target, os.O_RDWR|os.O_CREATE|os.O_EXCL, 0666)
+		if err != nil {
+			return fmt.Errorf("cannot dump to file: %w", err)
+		}
+		defer func() {
+			_ = file.Close()
+		}()
+
+		outputWriter = file
+	} else {
+		if err := checkStdoutArchive(); err != nil {
```

Comment:
That check previously only applied to cases in which `dump` would generate a tar/zip archive, but not when only outputting a single file. I'd like to keep it that way.

Reply:
Hey, 

Instead of creating a new Writer, I just updated the `os.Stdout` to the file. Because, if I used the `outputFileWriter` the `os.Stdout` would still be pointing to the Terminal.